proto/lookout/sdk/service_data.proto

```diff
@@ -37,44 +40,82 @@ message File {
     string path = 1;
     // POSIX-style file mode.
     uint32 mode = 2;
-    // Hash of the file contents.
+    // SHA1 hash of the file contents.
     string hash = 3;
-    // Raw content of the file.
+    // Raw content of the file. It will be empty if WantContents was not set in
+    // the request.
     bytes content = 4;
-    // UAST.
+    // UAST is a Babelfish v1 UAST of the file contents. It will be empty if
+    // WantUAST was not set in the request.
     gopkg.in.bblfsh.sdk.v1.uast.Node uast = 5 [(gogoproto.customname) = "UAST"];
-    // Programming/data/markup language of the file.
+    // Programming/data/markup language of the file as returned by enry. It will
+    // be empty if WantLanguage was not set in the request.
```

Comment:
The language will be also returned if `WantUAST` is set in the request.
```suggestion
    // be empty if none of WantLanguage or WantUAST was not set in the request.
```

Reply:
Done in 4a1b32b (I'll squash before merging)